### PR TITLE
add sqlite3 CI

### PR DESCRIPTION
**Pull request by borune-k12**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: .github/workflows/sqlite3.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+name: sqlite3 CI
+
+on:
+  pull_request:
+    paths:
+      - 'recipes/sqlite3/**'
```
Comment (worldemar) [inline .github/workflows/sqlite3.yml]:
```suggestion
      - 'recipes/sqlite3/all/**'
```

File: .github/workflows/sqlite3.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      - 'build.py'
+      - '.github/workflows/sqlite3.yml'
+  push:
+    branches:
+      - master
+    paths:
+      - 'recipes/sqlite3/**'
```
Comment (worldemar) [inline .github/workflows/sqlite3.yml]:
```suggestion
      - 'recipes/sqlite3/all/**'
```

File: .github/workflows/sqlite3.yml (hunk 1 above)
Comment (borune-k12) [reply .github/workflows/sqlite3.yml]: в ямле для openssl без ALL

Comment (worldemar) [reply .github/workflows/sqlite3.yml]: если yml собирает то что в папке `all`, то и триггер надо ставить туда

**Pull request merged**